openquake/commonlib/tests/calculators/__init__.py

```diff
@@ -36,21 +36,22 @@ def get_calc(self, testfile, job_ini):
         """
         self.testdir = os.path.dirname(testfile)
         inis = [os.path.join(self.testdir, ini) for ini in job_ini.split(',')]
-        oq = self.oqparam = readinput.get_oqparam(inis)
-        oq.concurrent_tasks = executor._max_workers
+        oq = readinput.get_oqparam(inis)
+        oq.concurrent_tasks = executor._max_workers * 4
```

Comment:
Perhaps this `4` might be transformed into a parameter (or you might add a comment)